config/checkstyle-checks.xml

```diff
@@ -613,6 +613,11 @@
       <property name="files" value=".*[\\/]src[\\/](test|it|xdocs-examples)[\\/]"/>
       <property name="checks" value="MethodCount"/>
     </module>
+    <module name="SuppressionXpathSingleFilter">
+      <property name="files"
+                value=".*[\\/]xpath[\\/]AbstractElementNode.java"/>
+      <property name="checks" value="ImportControl"/>
```

Comment:
I do not see any criminal to allow new import in this class.
Please remove suppression and add allowance for required import.

Reply:
done